smartprom.py

```diff
@@ -169,7 +274,11 @@ def collect():
     for drive, drive_attrs in DRIVES.items():
         typ = drive_attrs['type']
         try:
-            if typ in SAT_TYPES:
+            if "megaraid_id" in drive_attrs:
+                attrs = smart_megaraid(
+                    drive_attrs["megaraid_id"], drive_attrs["bus_device"]
```

Comment:
Update the readme with the information about MegaRAID. Add a new section and copy the examples from this issue.

Reply:
Added by https://github.com/matusnovak/prometheus-smartctl/pull/47/commits/22c53dfa4d9a1c1097aad969170b4441e78375e0